src/fiatconnect/constants.ts

```diff
@@ -0,0 +1,25 @@
+import { FiatAccountSchema } from '@fiatconnect/fiatconnect-types'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+
+export const CURRENCIES_WITH_FEE_DISCLAIMER: Partial<
+  Record<FiatAccountSchema, Set<LocalCurrencyCode>>
+> = {
+  [FiatAccountSchema.AccountNumber]: new Set([
```

Comment:
Should this list be in firebase? Otherwise we will have to wait a whole release cycle for the disclaimer to be in a new country thats supported.